service/api.go

```diff
@@ -765,15 +782,15 @@ func (as *APIServer) GetServiceApp() *gimlet.APIApp {
 	app.SimpleVersions = true
 
 	// Project lookup and validation routes
-	app.AddRoute("/ref/{identifier}").Handler(as.fetchProjectRef).Get()
-	app.AddRoute("/validate").Handler(as.validateProjectConfig).Post()
+	app.AddRoute("/ref/{identifier}").Wrap(requireUserToggleable).Handler(as.fetchProjectRef).Get()
+	app.AddRoute("/validate").Wrap(requireUserToggleable).Handler(as.validateProjectConfig).Post()
 
 	// Internal status reporting
-	app.AddRoute("/status/consistent_task_assignment").Handler(as.consistentTaskAssignment).Get()
-	app.AddRoute("/status/stuck_hosts").Handler(as.getStuckHosts).Get()
-	app.AddRoute("/status/info").Handler(as.serviceStatusSimple).Get()
-	app.AddRoute("/task_queue").Handler(as.getTaskQueueSizes).Get()
-	app.AddRoute("/task_queue/limit").Handler(as.checkTaskQueueSize).Get()
+	app.AddRoute("/status/consistent_task_assignment").Wrap(requireUserToggleable).Handler(as.consistentTaskAssignment).Get()
+	app.AddRoute("/status/stuck_hosts").Wrap(requireUserToggleable).Handler(as.getStuckHosts).Get()
+	app.AddRoute("/status/info").Wrap(requireUserToggleable).Handler(as.serviceStatusSimple).Get()
+	app.AddRoute("/task_queue").Wrap(requireUserToggleable).Handler(as.getTaskQueueSizes).Get()
+	app.AddRoute("/task_queue/limit").Wrap(requireUserToggleable).Handler(as.checkTaskQueueSize).Get()
```

Comment:
Can you check where these routes are used (since we use them as opposed to users) and verify that we pass in user information?

Reply:
Also should /dockerfile also have this flag?

Reply:
I've checked our project and others in our org and haven't found any invocation of any of these internal routes with the exception of `status/info`. They also don't show up in splunk history so I believe they should be unused and safe to remove.  

The `status/info` endpoint is invoked by the app servers setup scripts which don't pass user information. I'll remove the middleware for now and add a comment mentioning that we're currently leaving this route without middleware. Since this is a simple / low latency endpoint it worth to keep it this way or do we want to introduce a new ticket to work on passing user info from the python setup scripts that the app servers use?